packages/scene-composer/src/components/three-fiber/ModelRefComponent/__tests__/GLTFLoader.spec.ts

```diff
@@ -94,8 +94,7 @@ describe('GLTFLoader', () => {
     it('should execute without draco decoder enabled', async () => {
       const getGlobalSettingsMock = getGlobalSettings as jest.Mock;
       const dracoDecoder: DracoDecoderConfig = {
-        enable: true,
```

Comment:
The logic on this test was backwards & should have been testing with `dracoDecoder` disabled.